android/app/src/main/java/network/perun/app/MainActivity.java

```diff
@@ -52,8 +52,10 @@ protected void onCreate(Bundle savedInstanceState) {
             // in this case we already deployed them and enter their addresses.
             Address adjudicator = new Address("0xDc4A7e107aD6dBDA1870df34d70B51796BBd1335");
             Address assetholder = new Address("0xb051EAD0C6CC2f568166F8fEC4f07511B88678bA");
```

Comment:
Maybe camelcase for the asset holder?
```suggestion
            Address assetHolder = new Address("0xb051EAD0C6CC2f568166F8fEC4f07511B88678bA");
```